site/content/blogs/release-v127.en.md

```diff
@@ -0,0 +1,54 @@
+---
+title: 'AWS Copilot v1.27: Extend Copilot templates, additional routing rule supports, preview differences, and sidecar improvements!'
+twitter_title: 'AWS Copilot v1.27'
+image: ''
+image_alt: ''
+image_width: '1051'
+image_height: '747'
+---
+
+# AWS Copilot v1.27: Extend Copilot, preview differences, load balancer and sidecar improvements!
+##### Posted On: Mar 28, 2023
+
+The AWS Copilot core team is announcing the Copilot v1.27 release 🚀.  
+Our public [сommunity сhat](https://app.gitter.im/#/room/#aws_copilot-cli:gitter.im) is growing and has over 400 people online and over 2.6k stars on [GitHub](http://github.com/aws/copilot-cli/).
+Thanks to every one of you who shows love and support for AWS Copilot.
+
+Copilot v1.27 is a big release with several new features and improvements:
+
+- **Extend Copilot templates**: You can now customize any properties in Copilot-generated AWS CloudFormation templates 
+with the AWS Cloud Development Kit (CDK) or YAML Patch overrides. [See detailed section](#extend-copilot-generated-aws-cloudformation-templates).
+- **Enable multiple listener rules for Application Load Balancers**: You can define multiple host-based or path listener rules
+routing to different containers for [application load balancers](../docs/manifest/lb-web-service.en.md#http) associated with your Load Balanced Web or Backend Service. [See detailed section](#add-multiple-host-based-or-path-based-routing-rules-to-your-application-load-balancers).
+- **Enable multiple listeners for Network Load Balancers**: You can define multiple listeners on different ports and protocols
+  routing to different containers for [network load balancers](../docs/manifest/lb-web-service.en.md#nlb) associated with your Load Balanced Web Service. [See detailed section](#add-multiple-port-and-protocol-listeners-to-your-network-load-balancers).
```

Comment:
Should we just refer to `## Enable multiple listeners and routing rules for Load Balancers` instead of separating them into two items as they are very similar?